src/main/java/emu/grasscutter/command/commands/ClearCommand.java

```diff
@@ -61,43 +69,84 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
+                    }
+                }
+                case "all" -> {
+                    playerInventory.removeItems(getRelics(playerInventory, param).toList());
+                    CommandHandler.sendTranslatedMessage(sender, "commands.clear.artifacts", playerString);
+                    playerInventory.removeItems(getWeapons(playerInventory, param).toList());
+                    CommandHandler.sendTranslatedMessage(sender, "commands.clear.weapons", playerString);
+                    playerInventory.removeItems(
                         getOther(ItemType.ITEM_MATERIAL, playerInventory, param).toList());
-                CommandHandler.sendTranslatedMessage(sender, "commands.clear.materials", playerString);
-                playerInventory.removeItems(
+                    CommandHandler.sendTranslatedMessage(sender, "commands.clear.materials", playerString);
+                    playerInventory.removeItems(
                         getOther(ItemType.ITEM_FURNITURE, playerInventory, param).toList());
-                CommandHandler.sendTranslatedMessage(sender, "commands.clear.furniture", playerString);
-                playerInventory.removeItems(
+                    CommandHandler.sendTranslatedMessage(sender, "commands.clear.furniture", playerString);
+                    playerInventory.removeItems(
                         getOther(ItemType.ITEM_DISPLAY, playerInventory, param).toList());
-                CommandHandler.sendTranslatedMessage(sender, "commands.clear.displays", playerString);
-                playerInventory.removeItems(
+                    CommandHandler.sendTranslatedMessage(sender, "commands.clear.displays", playerString);
+                    playerInventory.removeItems(
                         getOther(ItemType.ITEM_VIRTUAL, playerInventory, param).toList());
-                CommandHandler.sendTranslatedMessage(sender, "commands.clear.virtuals", playerString);
-                CommandHandler.sendTranslatedMessage(sender, "commands.clear.everything", playerString);
+                    CommandHandler.sendTranslatedMessage(sender, "commands.clear.virtuals", playerString);
+                    CommandHandler.sendTranslatedMessage(sender, "commands.clear.everything", playerString);
+                }
+                default -> {
+                    try {
+                        param.id = Integer.parseInt(id);
+                    } catch (NumberFormatException e) {
+                        // TODO: Parse from item name using GM Handbook.
+                        CommandHandler.sendTranslatedMessage(sender, "commands.generic.invalid.itemId");
+                        throw e;
+                    }
+                    if (param.amount != -1) {
+                        playerInventory.removeItems(getId(param.id, playerInventory, param).toList().subList(0, param.amount));
+                        CommandHandler.sendTranslatedMessage(sender, "commands.clear.id", playerString, param.id, param.amount);
```

Comment:
Your new string uses the following template: "已清除 %s 的 **%s** 個 **%s**。"

```suggestion
                        CommandHandler.sendTranslatedMessage(sender, "commands.clear.id", playerString, param.amount, param.id);
```